pkg/receiver/gitproviderreceiver/internal/scraper/githubscraper/genqlient.graphql

```diff
@@ -106,3 +106,42 @@ query getBranchCount($name: String!, $owner: String!) {
     }
   }
 }
+
+query getPullRequestCount($name: String!, $owner: String!, $states: [PullRequestState!]) {
+  repository(name: $name, owner: $owner) {
+    pullRequests(states: $states){
+      totalCount
+    }
+  }
+}
+
+query getPullRequestData(
+  $name: String!
+  $owner: String!
+  $prFirst: Int!
+  # @genqlient(pointer: true)
+  $prCursor: String
+) {
+  repository(name: $name, owner: $owner) {
+    pullRequests(
+      first: $prFirst,
+      after: $prCursor,
+      states: [OPEN, MERGED]
```

Comment:
Since we're passing states into the count query, does it make to sense to use the same states variable in this query to ensure the same states are passed into the count & the `getPullRequestData` queries.